src/ts/uiutils.ts

```diff
@@ -21,4 +21,16 @@ export namespace UIUtils {
     // Walk and configure the component tree
     recursiveTreeWalker(component);
   }
+
+  // From: https://github.com/nfriend/ts-keycode-enum/blob/master/Key.enum.ts
+  export enum KeyCode {
+    LeftArrow = 37,
+    UpArrow = 38,
+    RightArrow = 39,
+    DownArrow = 40,
+
+    Space = 32,
+    End = 35,
```

Comment:
What is the `End` and `Home` key? 🤔

Reply:
Set the slider to the min and max value, required by the standard